bin/reth/src/args/rpc_server_args.rs

```diff
@@ -104,12 +105,13 @@ impl RpcServerArgs {
     }
 
     /// Convenience function for starting a rpc server with configs which extracted from cli args.
-    pub(crate) async fn start_rpc_server<Client, Pool, Network, Tasks>(
+    pub(crate) async fn start_rpc_server<Client, Pool, Network, Tasks, Events>(
         &self,
         client: Client,
         pool: Pool,
         network: Network,
         executor: Tasks,
+        events: Events,
     ) -> Result<RpcServerHandle, RpcError>
```

Comment:
These are starting to get larger and we should see if we can create better abstractions eg with a builder